lib/stripe/subscription_schedule.rb

```diff
@@ -9,6 +9,10 @@ class SubscriptionSchedule < APIResource
 
     OBJECT_NAME = "subscription_schedule".freeze
 
+    custom_method :cancel, http_verb: :post
+    custom_method :release, http_verb: :post
+    custom_method :revisions, http_verb: :get
```

Comment:
same as above, I don't think `revisions` should be static. We have `list_revisions` instead

Reply:
👍  `list_revisions` is already defined here, so we should just remove the `custom_method` line.

Reply:
removed